src/server/rpc/procedures/hawaii-co2-data/hawaii-co2-data.js

```diff
@@ -0,0 +1,103 @@
+const lineReader = require('line-reader');
+const path = require('path');
+
+const table = [
+    ['Year', 'Month', 'Interpolated ppm', 'Seasonally adjusted ppm']
+];
+const table1 = [
+    ['Year', 'Month', 'Interpolated ppm']
+];
+const table2 = [
+    ['Year', 'Month', 'Seasonally adjusted ppm']
+];
+
+
+lineReader.eachLine(path.join(__dirname,'co2_mm_mlo.txt'), function (line) {
+    if (!line.startsWith('#')) {
+        let year = line.substring(0, 4);
+        let month = line.substring(4, 8).trim();
+        let inter = line.substring(38, 44);
+        let season = line.substring(50, 56);
+        let wholeArr = [year, month, inter, season];
+        let interpolatedArr = [year, month, inter];
+        let seasonalArr = [year, month, season];
+        table.push(wholeArr);
+        table1.push(interpolatedArr);
+        table2.push(seasonalArr);
+    }});
+
+const co2service = {};
+
+co2service.serviceName = 'CO2Data';
```

Comment:
There might be a better name for the service (making the source of the data obvious as done in `IEXTrading`, `GoogleMaps`, etc). With the exception of correcting capitalization (which you will likely need to do for `CO2`), this name should match the directory and filename.

Check out https://github.com/NetsBlox/NetsBlox/wiki/Best-Practices-for-NetsBlox-Services#naming-conventions